package lesson04.part03;
//@FoLoKe
import org.junit.Test;
import org.junit.runner.RunWith;
import org.junit.runners.JUnit4;

import static org.junit.Assert.*;

@RunWith(JUnit4.class)
public class Task45Test {
    /* 1.	Метод main должен вызывать метод method1.
     * 2.	Метод main должен перехватывать исключение NullPointerException.
     * 3.	Метод main должен перехватывать исключение FileNotFoundException.
     * 4.	Метод main не должен перехватывать исключение ArithmeticException.
     * 5.	Метод main не должен перехватывать исключение URISyntaxException.
     * 6.	Метод method1 не изменять.
     */

    @Test
    public void test45main() {

    }

    @Test
    public void test4501() {

    }

    @Test
    public void test4502() {

    }

    @Test
    public void test4503() {

    }

    @Test
    public void test4504() {

    }
}